src/Flexible.php

```diff
@@ -261,11 +263,11 @@ public function resolveForDisplay($resource, $attribute = null)
     {
         $attribute = $attribute ?? $this->attribute;
 
-        $this->registerOriginModel($resource);
-
-        $this->buildGroups($resource, $attribute);
-
-        $this->value = $this->resolveGroupsForDisplay($this->groups);
+        if (! $this->displayCallback) {
+            $this->resolve($resource, $attribute);
+        } elseif (is_callable($this->displayCallback)) {
```

Comment:
Same thing here about returning early

Reply:
I'll change it to just a normal return void. The Field class documents it should return a void. Don't want to overwrite that and create any unnecessary side-effects.

Reply:
Even better 👍